rebar.config

```
@@ -10,7 +10,7 @@
 {xref_checks,[undefined_function_calls,undefined_functions,locals_not_used,
     deprecated_function_calls, deprecated_functions]}.
 
-{plugins, [{rebar3_eqc, {git, "https://github.com/Vagabond/rebar3-eqc-plugin", {branch, "master"}}}]}.
+{plugins, [rebar3_eqc]}.
```

Sorry, my mistake. It turns out we really need Vagabond's version
```suggestion
{plugins, [{rebar3_eqc, {git, "https://github.com/Vagabond/rebar3-eqc-plugin", {branch, "master"}}}]}.
```